.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parserOptions": {
+    "createDefaultProgram": true,
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "plugins": ["prettier"],
+  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "rules": {
+    "no-eval": 2,
+    "prefer-const": 2
+  }
+}
```

Comment:
please update with *eslintconfig* from issue description

Reply:
Updated in d3482ab3cd3bc0279ad198369012a9fd8b289138